addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -2266,7 +2286,7 @@ export class Wysiwyg extends Component {
 
         return finalOptions;
     }
-    _getBannerCommand(title, alertClass, iconClass, description, priority) {
+    _getBannerCommand(title, emoji, alertClass, iconClass, description, priority) {
```

Comment:
I guess we still need the "iconClass" for the command when displayed inside the powerbox?

Reply:
Yes, that's it :smile: